### PR TITLE
RELATED: CL-9029 Execute and render bar chart

### PR DESCRIPTION
**Pull request by dprentis**

### CODE REVIEW

**Comment by yenkins-admin** [comment]
Test FAILed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/126/

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/127/

File: src/execution.js [hunk 1]
```diff
@@ -352,10 +358,12 @@ export const mdToExecutionConfiguration = (mdObj) => {
         flatten(contributionPoPMetrics)
     );
 
+    const orderBy = map(filter(allMetrics, item => !!item.sort), sortToOrderBy);
     const where = [].concat(attributeFilters, dateFilters).reduce(assign, {});
 
     return { execution: {
         columns: filter(map(allMetrics, 'element'), identity),
+        orderBy,
```
Comment (vaclavbohac) [inline src/execution.js]: Add simple test for this

Comment (dprentis) [inline src/execution.js]: It is done

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/135/

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/136/

**Pull request merged**